deis/cloud-config.yaml

```diff
@@ -0,0 +1,178 @@
+#cloud-config
+---
+coreos:
+  etcd:
+    # generate a new token for each unique cluster from https://discovery.etcd.io/new
+    # uncomment the following line and replace it with your discovery URL
+    discovery: https://discovery.etcd.io/402d2ffba1539292992aff540c8dc3d9
```

Comment:
the yaml must be generated at provision time. Discovery urls are only good to use once per cluster. see:
https://github.com/deis/deis/blob/master/contrib/azure/create-azure-user-data